### DISCUSSION TITLE
How to use mathjax in the chirpy-starter template

### DISCUSSION THREAD

**Discussion by hleecaster** [cotes2020/jekyll-theme-chirpy#1530] [Q&A] [answered]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

I created a site using the [chirpy-starter](https://github.com/cotes2020/chirpy-starter) template.
And I want to use mathjax syntax, but it's not working.
Is there any way to apply mathjax to the chirpy-starter template?

### Operations you have already tried

All I did was modify the `_config.yml` file when I created the site.

**Comment by NichtsHsu** [comment, marked as answer]
Before asking a question, you should check all available documentation: https://chirpy.cotes.page/posts/write-a-new-post/#mathematics

**Reply by hleecaster** [reply]
Sorry. It's my fault. I didn't check carefully enough.

I hope @cotes2020 will delete this discussion.
